generators/server/needle-api/needle-server-liquibase.js

```diff
@@ -48,6 +48,13 @@ module.exports = class extends needleServer {
         this.addBlockContentToFile(rewriteFileModel, errorMessage);
     }
 
+    addLoadColumnToEntityChangeSet(filePath, content) {
```

Comment:
I would have designed this API differently, e.g. `addLoadColumnToEntityChangeSet(filePath, columnName, otherAttributes)` where otherAttributes whould be a map  and then would have filled a template `<column name="${columnName}" attribute1="..." attribute2="..." />` in the spirit of what is done for the maven needle api. 

But what you've done is consistent with the existing API of needle-server-liquibase